.vcpkg-overlays/ports/dune-alugrid/vcpkg.json

```diff
@@ -15,5 +15,15 @@
     {
       "name": "dune-grid"
     }
-  ]
+  ],
+  "features": {
+    "uggrid": {
```

Comment:
Why is UG a feature of dune-alugrid?

Reply:
Stuff breaks otherwise :)
It's an order thing, if alugrid is configured before UG, then stuff breaks. 
I _think_ this might be due to `dune.module` parsing and suggested deps there.